algebraic-graphs.cabal

```diff
@@ -66,6 +66,7 @@ source-repository head
 library
     hs-source-dirs:     src
     exposed-modules:    Algebra.Graph,
+                        Algebra.Graph.Undirected.UndirectedGraph,
```

Comment:
This should be just `Algebra.Graph.Undirected`.

Reply:
Got it!